.github/workflows/l10n.yml

```diff
@@ -0,0 +1,27 @@
+name: daily l10n run
+
+on:
+  schedule:
+    - cron: "0 0 * * 1-5"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.head_ref }}
+
+      # Runs a set of commands using the runners shell
+      - name: run l10n scripts
+        run: |
+          cd Composer
+          yarn l10n
+
+      - uses: stefanzweifel/git-auto-commit-action@v4
+        with:
+          commit_message: find and localize new strings
+          commit_author: Localization Bot c/o Ben Yackley <beyackle@microsoft.com>
```

Comment:
Consider using the `file_patterns` option here to only commit files we expect to have changed.